js/web/lib/wasm/jsep/webgpu/ops/transpose.ts

```diff
@@ -13,14 +13,18 @@ export interface TransposeAttributes extends AttributeWithCacheKey {
   readonly perm: number[];
 }
 
-const validateInputs = (inputs: readonly TensorView[]): void => {
+const validateInputs = (inputs: readonly TensorView[], perm: number[]): void => {
   if (!inputs || inputs.length !== 1) {
     throw new Error('Transpose requires 1 input.');
   }
+
+  if (perm.length != 0 && perm.length !== inputs[0].dims.length) {
```

Comment:
use `!==` instead of `!=`, which may get into unexpected result in JavaScript.

Reply:
Done